crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -58,6 +58,8 @@ pub struct DaemonServer<T: Watcher> {
     shutdown_rx: Option<Receiver<()>>,
 
     running: Arc<AtomicBool>,
+
+    times_saved: HashMap<String, u64>,
```

Comment:
We want to wrap the times saved map in `Arc<Mutex<T>>` to allow for us to mutate this from different requests. 
Breaking it down:
 - `Arc` is "atomically reference counted" extremely similar to `Rc` which stands for reference counted which changes a value from being fully dropped whenever it goes out of scope to just decrementing the count and having the value get dropped once it hits 0. [`Arc` is thread safe unlike `Rc`](https://doc.rust-lang.org/std/sync/struct.Arc.html#thread-safety)
 - Mutex: just a normal mutex
```suggestion
    times_saved: Arc<std::sync::Mutex<HashMap<String, u64>>>,
```